tutor/src/components/course-calendar/add-assignment-sidebar.cjsx

```diff
@@ -1,43 +1,90 @@
 React = require 'react'
 BS = require 'react-bootstrap'
+isEmpty = require 'lodash/isEmpty'
 
 classnames = require 'classnames'
+{UiSettings} = require 'shared'
 
 {AddAssignmentLink} = require './task-dnd'
 
 PastAssignments = require './past-assignments'
 
 CourseAddMenuMixin = require './add-menu-mixin'
+BindStoreMixin = require '../bind-store-mixin'
+
 {CourseStore} = require '../../flux/course'
+{PastTaskPlansStore} = require '../../flux/past-task-plans'
+
+IS_INTRO_VIEWED = 'viewed-plan-dnd-intro'
+USE_SETTINGS = false
 
 AddAssignmentSidebar = React.createClass
 
-  mixins: [ CourseAddMenuMixin ]
+  mixins: [ CourseAddMenuMixin, BindStoreMixin ]
+  bindStore: PastTaskPlansStore
 
   propTypes:
     courseId: React.PropTypes.string.isRequired
     hasPeriods: React.PropTypes.bool.isRequired
-
+    isOpen: React.PropTypes.bool.isRequired
 
   getInitialState: ->
-    {isOpen: false}
+    showIntro: @props.isOpen
+    needsIntro: false
+
```

Comment:
The intro never seems to show for me, because the calendar loads with `isOpen` set to false causing `showIntro` to be set to false as well here.  Then when the `shouldShowIntro` method is called below the last clause that checks `showIntro` is always false and fails.